src/include/gen/msgs.h

```diff
@@ -968,6 +968,7 @@ Data source : @4"},		/* eds_statement */
 	{335545261, "Can not convert @1 to @2"},		/* bind_convert */
 	{335545262, "cannot update old BLOB"},		/* cannot_update_old_blob */
 	{335545263, "cannot read from new BLOB"},		/* cannot_read_new_blob */
+	{335545264, "There is no privilege CREATE @1"},		/* dyn_no_create_priv */
```

Comment:
"User has no CREATE @1 privilege" ?
"Privilege CREATE @1 is not granted to user" ?
"No permission for CREATE @1 operation" ?

Reply:
User has no CREATE @1 privilege